src/lib/hooks/useWindowSize.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+
+// taken from https://usehooks.com/useWindowSize/
+export const useWindowSize = () => {
```

Comment:
I didn't end up using this, but I think this could be a useful hook so I'm going to leave it in there